ui/public/index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <script src="../env-config.js"></script>
```

Comment:
Would `<%= BASE_URL %>../env-config.js` work? Not specifying `BASE_URL` can cause issues with e2e tests if I'm not mistaken.

Reply:
sooo, this file is not even being served in dev environment right now